### ISSUE TITLE
Unable to find a specification for 'RSKImageCropper' depended upon by 'RNImageCropPicker'

### ISSUE THREAD

**Issue by 1512751361** [ivpusic/react-native-image-crop-picker#1084]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.25.0
- react-native v0.64.0

### Platform
Tell us to which platform this issue is related

- iOS
pod install
![{B0CC7B4B-1BBE-63E2-BBCF-47AB8F7DDFA2}](https://user-images.githubusercontent.com/24776858/62101400-eca54180-b2c7-11e9-850f-78c849ca043e.png)

- Android

### Steps to reproduce

1. ios工程，pod install 的时候报错

2.

3.

### Attachments

// stacktrace or any other useful debug info

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by karltaylor** [comment]
I also have this issue however I am using a different fork.

```
[!] CocoaPods could not find compatible versions for pod "RSKImageCropper":
  In snapshot (Podfile.lock):
    RSKImageCropper (= 2.2.3)
  In Podfile:
    RNImageCropPicker (from `../node_modules/react-native-image-crop-picker`) was resolved to 0.25.0, which depends on
      RSKImageCropper

```

**Comment by ivpusic** [comment]
try `pod repo update` and than `pod install`
